common/Cargo.toml

```diff
@@ -6,11 +6,11 @@ edition = "2021"
 # client
 [dependencies.substrate-api-client]
 git = "https://github.com/Cardinal-Cryptography/substrate-api-client.git"
-tag = "fixed_dependencies"
+tag = "polkadot-v0.9.13"
```

Comment:
it is strange, that this tag does not refer to a commit within the repo

Reply:
resolved offline